apollo/__main__.py

```diff
@@ -110,6 +110,9 @@ def add_template_args(my_parser, default_template):
     source2bags_parser.add_argument(
         "--min-docfreq", default=1, type=int,
         help="The minimum document frequency of each element.")
+    source2bags_parser.add_argument(
+        "--nb-partitions", default=10, type=int,
+        help="Target number of partitions in the quantization.")
```

Comment:
Let's use the terminology from physics: "number of quantization levels"

Reply:
Done